app/assets/javascripts/utilities/buildCommentFormHTML.js.erb

```diff
@@ -18,7 +18,7 @@ function buildCommentFormHTML(commentableId, commentableType, parentId) {
       <input value="${parentId}" type="hidden" name="comment[parent_id]" id="comment_parent_id" />
       <div class="comment-form__inner">
         <div class="comment-form__field">
-          <textarea id="textarea-for-${parentId}" class="crayons-textfield crayons-textfield--ghost comment-textarea" name="comment[body_markdown]" placeholder="Reply..." aria-label="Reply to a comment..." required="required" onkeydown="handleKeyDown(event)" onfocus="handleFocus(event)" oninput="handleChange(event)" onkeykup="handleKeyUp(event)"></textarea>
+          <textarea id="textarea-for-${parentId}" class="crayons-textfield crayons-textfield--ghost comment-textarea" name="comment[body_markdown]" placeholder="Reply..." aria-label="Reply to a comment..." required="required" onkeydown="handleKeyDown(event)" onfocus="handleFocus(event)" oninput="handleChange(event)" onkeyup="handleKeyUp(event)"></textarea>
```

Comment:
Thanks for all the work in this PR @jsoref! Since this is quite a bit and hard to review PR, could you maybe break it down into smaller ones? I'd really like to get some things that are actual bugs (like this one) out faster.

Reply:
Sure, just let me know how you want it split.

Reply:
@jsoref The most important one to me would be everything that is 

1. actual code, not a comment (`currrentTime`, `onkeykup`, etc.)
2. not in a migration

Once we have that out of the way we can think about how to slice and dice the rest.